system-tests/projects/firefox-memory/cypress/plugins/index.js

```diff
@@ -0,0 +1,111 @@
+// ***********************************************************
```

Comment:
The contents of this file are moved to `setupNodeEvents` in 10.0-release: https://github.com/cypress-io/cypress/blob/10.0-release/system-tests/projects/firefox-memory/cypress.config.js#L13

This is the only relevant change: https://github.com/cypress-io/cypress/commit/731c7febd1d130a7bd0a8d7ac863eecb1a62c648#diff-2bd39bf99582141da07f09ca30d61dd6fd83b597dcc3c8fc2abc37a7f773f89d